docs/started/daemons.md

```diff
@@ -0,0 +1,92 @@
+---
+id: daemons
+title: Rucio Daemons
+sidebar_label: daemons
+---
+
+Rucio relies on several daemons (processes) to perform different logic.
+Most of the daemons connect to the DB to read some data, perform some computation, 
+and then write some data back into the DB.
+
+Usually one daemon will create some work for another daemon and viceversa.
+In Rucio realm, daemons communicate to others by the DB.
+
+The following table reprensents a high level view of the responsibility of each of the daemons.
+
+## Daemons
+
+Name | Domain|  Purpose |  Details
+-----------| ----------------- | -------| ----- |
+rucio-abacus-account | Accounting | Account usage | [Details](https://rucio.cern.ch/documentation/bin/rucio-abacus-account)
+rucio-abacus-collection-replica | Accounting |  Updates collection replicas | [Details](https://rucio.cern.ch/documentation/bin/rucio-abacus-collection-replica)
+rucio-abacus-rse | Accounting | Updates RSE counters | [Details](https://rucio.cern.ch/documentation/bin/rucio-abacus-rse)
+rucio-atropos | Replica | End the life of the rules according to the Lifetime Model | [Details](https://rucio.cern.ch/documentation/bin/rucio-atropos)
+rucio-auditor | Replica | Find inconsistencies on storage, for example, dark data discovery | [Details](https://rucio.cern.ch/documentation/bin/rucio-auditor)
+rucio-automatix | Replica | Used for testing: injects random data in RSEs to check liveness | [Details](https://rucio.cern.ch/documentation/bin/rucio-automatix)
+rucio-bb8 | Replica | Rebalance data across RSEs | [Details](https://rucio.cern.ch/documentation/bin/rucio-bb8)
+rucio-c3po | Replica | Places data according to dynamic rules: available space, popularity, etc ... | [Details](https://rucio.cern.ch/documentation/bin/rucio-c3po)
+rucio-cache-client | Replica | Populates information of replicas on volatile storage  | [Details](https://rucio.cern.ch/documentation/bin/rucio-cache-client)
+rucio-cache-consumer | Replica | Adds and deletes cache replicas to the Rucio catalog | [Details](https://rucio.cern.ch/documentation/bin/rucio-cache-consumer)
+rucio-conveyor-finisher | Transfer | Updates Rucio internal state after the file transfer has finished | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-finisher)
+rucio-conveyor-poller | Transfer | Polls updates from the transfer tool to check the transfer state | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-poller)
+rucio-conveyor-preparer | Transer | Prepares data transfers | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-preparer)
+rucio-conveyor-receiver | Transfer | Sister of polller, instead of polling for updates, it reads transfer tools notifications to check transfer state | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-receiver)
```

Comment:
```suggestion
rucio-conveyor-receiver | Transfer | Sister of poller, instead of polling for updates, it reads transfer tools notifications to check transfer state | [Details](https://rucio.cern.ch/documentation/bin/rucio-conveyor-receiver)
```

Could be nicer to say "similar to" or "companion to" instead of "sister of"